src/napari_time_slicer/TimelapseConverter.py

```diff
@@ -0,0 +1,320 @@
+import numpy as np
+from napari.layers import Points, Surface, Image, Labels, Vectors, Layer
```

Comment:
Would it be possible to use delayed imports here?
Otherwise this line could cause this bug again: https://github.com/haesleinhuepf/napari-segment-blobs-and-things-with-membranes/issues/12

Reply:
Changed in a07ea618d17ddaf1481cd7b29c0313504359105e